### PR TITLE
SW-1850 View/Edit Nursery

### PR DESCRIPTION
**Pull request by constanzauanini**
View:
Screen Shot 2022-10-14 at 16 25 01

Edit:
Screen Shot 2022-10-14 at 16 25 09

### CODE REVIEW

File: src/components/NewNursery/index.tsx [hunk 1]
```diff
@@ -57,13 +76,24 @@ export default function NurseryView({ organization, reloadOrganizationData }: Si
...
     } else {
-      snackbar.toastError();
+      const response = await createFacility(record);
+      if (response.requestSucceeded) {
+        reloadOrganizationData();
+        goToNurseries();
+        snackbar.toastSuccess(strings.NURSERY_ADDED);
+      } else {
+        snackbar.toastError();
+      }
```
Comment (karthikbtf) [inline src/components/NewNursery/index.tsx:96]: just a suggestion to refactor:
```
const response = selectedNursery 
  ? await updateFacility({ ...record, id: selectedNursery.id } as Facility)
  :  await createFacility(record);
if (response.requestSucceeded) {
  snackbar.toastSuccess(selectedNursery ? strings.CHANGES_SAVED : strings.NURSERY_ADDED);
  goToNurseries();
} else {
  toastError();
}
```

**Review by karthikbtf — APPROVED** [review]
looks good, one suggestion.. also if you can check it works on mobile

**Comment by constanzauanini** [comment]
> looks good, one suggestion.. also if you can check it works on mobile

Mobile:

Screen Shot 2022-10-14 at 16 57 32
Screen Shot 2022-10-14 at 16 57 47

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**